base/applications/mspaint/lang/it-IT.rc

```diff
@@ -15,23 +16,23 @@ BEGIN
     POPUP "&File"
     BEGIN
         MENUITEM "Nuovo\tCtrl+N", IDM_FILENEW
```

Comment:
I do assume the Nuovo should have an accel as well?

Reply:
Indeed, it's the N.